pinot-core/src/main/java/org/apache/pinot/core/operator/transform/function/TransformFunctionFactory.java

```diff
@@ -59,6 +59,10 @@
  * Factory class for transformation functions.
  */
 public class TransformFunctionFactory {
+
+  private static final String AGGREGATE_GAP_FILL = "aggregategapfill";
```

Comment:
Any specific reason naming it `aggregategapfill` instead of `gapfill`?

Reply:
aggregategapfill is for post-aggregation gapfill, meaning aggregation, then gapfill
We will use gapfillaggregate for pre-aggregate gapfill, meaning gapfill and then aggregate.